docs/notebooks/flax_basics.ipynb

```diff
@@ -903,13 +891,37 @@
     },
     {
       "cell_type": "code",
+      "execution_count": 22,
       "metadata": {
-        "id": "J6_tR-nPzB1i",
         "colab": {
           "base_uri": "https://localhost:8080/"
         },
+        "id": "J6_tR-nPzB1i",
         "outputId": "75465fd6-cdc8-497c-a3ec-7f709b5dde7a"
       },
+      "outputs": [
```

Comment:
The notebooks are ran by the docs built. So it's best to save them in Git without the output blocks.

Reply:
The output blocks were already there, I didn't add them. It looks like somehow they have been moved around, e.g. this output block you highlighted is actually at line 939 in the left side of the diff.

Reply:
(note though that when opening the notebook on Colab the outputs are only visible when stored with outputs; depending on the notebook it can be useful to have the outputs in Colab already precomputed or not...)

Reply:
> note though that when opening the notebook on Colab the outputs are only visible when stored with outputs

Didn't think of that. Ignore me :)